telegraf.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) {|f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'influxdb', '~> 0.3.15'
+  spec.add_dependency 'influxdb', '~> 0.3'
```

Comment:
I cannot reliable depend on `~> 0.x` because there can be breaking changes with each `0.x` version (and there are according to the changelog).

But I'm fine with having the version constraint removed completely and mentioning that in the README. Only when influx releases a `1.x` version I can add a looser constraint according to semver.